### PR TITLE
DOCAPI-6423: simpleLinearRegression function docs

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Documentation for #4917

### CODE REVIEW

File: docs/ru/getting_started/example_datasets/ontime.md [hunk 1]
```diff
@@ -193,7 +193,7 @@ SELECT Carrier, count(*) FROM ontime WHERE DepDelay>10 AND Year = 2007 GROUP BY
 Q5. Процент задержек по перевозчикам за 2007 год
 
 ``` sql
-SELECT Carrier, c, c2, c*1000/c2 as c3
+SELECT Carrier, c, c2, c*100/c2 as c3
```
Comment (blinkov) [inline docs/ru/getting_started/example_datasets/ontime.md]: let's make it consistent: we either have spaces around math operators or we don't

File: docs/ru/query_language/agg_functions/reference.md [hunk 2]
```diff
@@ -482,4 +482,44 @@ FROM ontime
...
+```
+
+Параметры:
+
+- `x` — столбец со значениями зависимой переменной.
+- `y` — столбец со значениями наблюдаемой переменной.
+
+Возвращаемые значения:
+
+Параметры `(a, b)` результирующей прямой `x = a*y + b`.
```
Comment (blinkov) [inline docs/ru/query_language/agg_functions/reference.md]: параметры прямой путаются с параметрами функции, лучше придумать синоним

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/ru/getting_started/example_datasets/ontime.md (hunk 1 above)
Comment (BayoNet) [reply docs/ru/getting_started/example_datasets/ontime.md]: I've deleted spaces around multiplication sign in the file, but I don't know how to be consistent in this question.
Compare `x = a * y + b` and `x = a*y + b`. I'd prefer the second variant, but it is not consistent.

File: docs/en/getting_started/example_datasets/ontime.md [hunk 3]
```diff
@@ -219,7 +219,7 @@ ORDER BY c3 DESC;
 Better version of the same query:
 
 ``` sql
-SELECT Carrier, avg(DepDelay > 10) * 100 AS c3 FROM ontime WHERE Year = 2007 GROUP BY Carrier ORDER BY Carrier
+SELECT Carrier, avg(DepDelay > 10)*100 AS c3 FROM ontime WHERE Year = 2007 GROUP BY Carrier ORDER BY Carrier
```
Comment (blinkov) [inline docs/en/getting_started/example_datasets/ontime.md]: if we prefer that there're no spaces around binary operators here then we should remove them around `>` and `=` (though usually I personally to have them), in many places below as well

P.S. also I think these queries would look better if we add some newlines before major keywords

File: docs/ru/getting_started/example_datasets/ontime.md [hunk 4]
```diff
@@ -251,7 +251,11 @@ ORDER BY c3 DESC;
 Более оптимальная версия того же запроса:
 
 ``` sql
+<<<<<<< HEAD
```
Comment (blinkov) [inline docs/ru/getting_started/example_datasets/ontime.md]: some merge artifats

File: docs/ru/getting_started/example_datasets/ontime.md [hunk 5]
```diff
@@ -219,7 +219,7 @@ ORDER BY c3 DESC;
 Более оптимальная версия того же запроса:
 
 ``` sql
-SELECT Carrier, avg(DepDelay > 10) * 100 AS c3 FROM ontime WHERE Year = 2007 GROUP BY Carrier ORDER BY Carrier
+SELECT Carrier, avg(DepDelay > 10)*100 AS c3 FROM ontime WHERE Year = 2007 GROUP BY Carrier ORDER BY Carrier
```
Comment (blinkov) [inline docs/ru/getting_started/example_datasets/ontime.md]: style

**Review by blinkov — CHANGES_REQUESTED** [review]

**Comment by BayoNet** [comment]
Здесь вроде бы всё хорошо. Вмёржить бы :-)

**Pull request merged**